webtau-rest/src/main/java/com/twosigma/webtau/http/HttpUrl.java

```diff
@@ -42,6 +46,34 @@ public static String extractPath(String url) {
         }
     }
 
+    public static Map<String, List<String>> extractQueryParams(String url) {
```

Comment:
can we make it return empty map instead of null?

Reply:
Yes